hw/ip/pwrmgr/doc/theory_of_operation.md

```diff
@@ -196,13 +196,11 @@ Once a steady state is reached, reset requests are then observed and processed.
 
 ### Reset Recording
 
-There are three ways in which the device is reset:
-- Non-debug-module reset request
+There are two ways in which the device is reset:
+- Reset requests mentioned in [reset handling](#reset-request-handling)
```

Comment:
Maybe this would be better as something like "Reset requests, as mentioned in..."? Otherwise, a reader (me!) might initially think that reset requests from that document are one way, but other types of reset requests are another way! :-)

Reply:
The other type of reset is mentioned below. I think saying "There are two ways in ..." makes it clear the two ways are as described. I made a minor change in words that makes the first item more concrete.

Reply:
Thanks, that's much clearer (to me)!